src/controllers/question.js

```diff
@@ -10,24 +10,40 @@ export async function get(ctx) {
 }
 
 export async function checkAnswer(ctx) {
+  //var wrongAttempts=0
+  var correctAttempts=0
   const { qno } = ctx.params
   const question = await Question.findOne({
     where: { qno },
   })
   ctx.body = ctx.request.body;
   const {user} = ctx.state
+  const preAnswered = await UserAnswer.findAll({
+    where:{
+      questionId: question.id,
+      userId: user.id 
+    },
+  })
+  for(var i in preAnswered){
+    if (preAnswered[i].useranswer in JSON.parse(question.answer))
```

Comment:
Store the output of JSON.parse(question.answer) in a variable outside the for loop. It will avoid recomputing it everytime.